### DISCUSSION TITLE
Change logo in bootscore 6

### DISCUSSION THREAD

**Discussion by chopinesque** [bootscore/bootscore#891] [Q&A] [answered]
I am trying to change the logo to a png or jpg file but I cannot find any way to do it. Using the child theme.

**Comment by crftwrk** [comment, marked as answer]
Use a filter in child's `functions.php` https://bootscore.me/documentation/filters/#change-paths-to-logos:

```php
/**
 * Change path to logos
 */
function change_logo_path($logo, $color) {
  if ($color === 'theme-dark') {
    return get_stylesheet_directory_uri() . '/path/to/dark-logo.png';
  }
  return get_stylesheet_directory_uri() . '/path/to/default-logo.png';
}
add_filter('bootscore/logo', 'change_logo_path', 10, 2);
```

Solved?

**Comment by chopinesque** [comment]
Wow, amazingly fast support! My compliments!

**Reply by crftwrk** [reply]
Thank you! I reopen this discussion because other users may have the same question.
